lib/model/CStringStore.cc

```diff
@@ -169,7 +181,7 @@ std::size_t CStringStore::memoryUsage() const {
     // core::CStoredStringPtr::dynamicSizeAlwaysZero() combined with dead code
     // elimination will make calculating the size of m_Strings boil down to a
     // couple of simple multiplications and additions
-    mem += core::memory::dynamicSize(m_Strings);
+    // mem += core::memory::dynamicSize(m_Strings);
```

Comment:
Commenting out this returned [this terrible performance](https://github.com/elastic/ml-cpp/pull/2650/files#r1549841469) to normal